### ISSUE TITLE
Roof removal: Stop it from working through interfaces

### ISSUE THREAD

**Issue by hugofaas** [runelite/runelite#13886]
Currently, the Roof removal plugin works through the inventory. In the image inventory on the left I have the mouse at the bottom, in the inventory on the right I have it at the top.

I don't think there's a situation where that makes sense, you're not trying to move into a building or look into a building when your mouse is hovering the inventory.
Same thing for the other right-bottom interfaces + the world map. I would keep it for the transparent chatbox.

It's a bit annoying to look at, to have big graphical changes (hiding / unhiding a roof) while clicking things in your inventory, so it would be nice if the plugin could be prevented from working through interfaces.

![roof removal](https://user-images.githubusercontent.com/1769781/125910754-edd005f1-4ef9-40e2-95b2-276d926cd7e4.png)

**Comment by Nightfirecat** [comment]
I could understand this being prevented while hovering the inventory, but you're able to click through the transparent chatbox aren't you?

**Comment by hugofaas** [comment]
Yes, hence the "I would keep it for the transparent chatbox." :)
With that I meant keeping the functionality through the chatbox.

**Comment by neeerp** [comment]
I'd like to also add that this is relevant to all interfaces and not just the map and inventory; for instance the collection log, game settings, combat achievement windows, etc... It's not a huge impact however it is a bit distracting.

If someone's working on this already, do mention it! Otherwise I might try to take a look if I can find time this week (not a promise).
